airflow/config_templates/config.yml

```diff
@@ -28,6 +28,14 @@ core:
       type: string
       example: ~
       default: "{AIRFLOW_HOME}/dags"
+    dag_bundle_storage_path:
```

Comment:
wonder if it will want its own section, or dag processing section, or something like that. just a wonder.

Reply:
Maybe. Going to table this for later.